src/Resources/public/js/fragmentList.jquery.js

```diff
@@ -247,15 +251,18 @@
         /**
          * This method cancels the fragment deletion.
          *
-         * @param {event} e
+         * @param {Plugin} plugin
          */
-        cancelFragmentDeletion: function (e) {
+        cancelFragmentDeletion: (plugin) => function (e) {
```

Comment:
I'm not really into javascript, but is this syntax fine? (compared with the existing one in this file)

Reply:
Indeed, @srascar Can you write this with an older syntax for compatibility ?
```
function (plugin) {
     return function (e) {
          ...
     }
}
```
I think

Reply:
Correct. Done